### PR TITLE
Added library checking support for red hat distros

### PR DESCRIPTION
**Pull request by fishfishfish2104**
library checking for red hat distros was only tested on fedora but should work on most using yum. to add more distros add them to the or statement on this [line](https://github.com/fishfishfish2104/wails/blob/f9a18817b74828ff8fae92a1a873087f9adc363b/cmd/linux.go#L81)

Changed some Package help inconsistencies

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,6 +12,7 @@ require (
 	github.com/leaanthony/mewn v0.10.5
 	github.com/leaanthony/slicer v1.3.1
 	github.com/leaanthony/spinner v0.5.0
+	github.com/matishsiao/goInfo v0.0.0-20170803142006-617e6440957e
```
Comment (leaanthony) [inline go.mod]: I think this is unused?
```suggestion
```

File: go.sum [hunk 2]
```diff
@@ -31,6 +31,8 @@ github.com/leaanthony/synx v0.1.0 h1:R0lmg2w6VMb8XcotOwAe5DLyzwjLrskNkwU7LLWsyL8
 github.com/leaanthony/synx v0.1.0/go.mod h1:Iz7eybeeG8bdq640iR+CwYb8p+9EOsgMWghkSRyZcqs=
 github.com/leaanthony/wincursor v0.1.0 h1:Dsyp68QcF5cCs65AMBmxoYNEm0n8K7mMchG6a8fYxf8=
 github.com/leaanthony/wincursor v0.1.0/go.mod h1:7TVwwrzSH/2Y9gLOGH+VhA+bZhoWXBRgbGNTMk+yimE=
+github.com/matishsiao/goInfo v0.0.0-20170803142006-617e6440957e h1:Y+GY+bv5vf1gssphFsGiq6R8qdHxnpDZvYljFnXfhD8=
```
Comment (leaanthony) [inline go.sum]: I think this is unused?
```suggestion
```

File: go.sum [hunk 3]
```diff
@@ -31,6 +31,8 @@ github.com/leaanthony/synx v0.1.0 h1:R0lmg2w6VMb8XcotOwAe5DLyzwjLrskNkwU7LLWsyL8
...
+github.com/matishsiao/goInfo v0.0.0-20170803142006-617e6440957e/go.mod h1:yLZrFIhv+Z20hxHvcZpEyKVQp9HMsOJkXAxx7yDqtvg=
```
Comment (leaanthony) [inline go.sum]: I think this is unused?
```suggestion
```

File: cmd/system.go [hunk 4]
```diff
@@ -283,6 +283,18 @@ func CheckDependencies(logger *Logger) (bool, error) {
...
 				}
+			case RedHat:
+
+				installed, err := YumInstalled(library.Name)
+				if err != nil {
+					return false, err
+				}
+				if !installed {
+					errors = true
+					logger.Red("Library '%s' not found. %s", library.Name, library.Help)
```
Comment (leaanthony) [inline cmd/system.go]: Could you please use logger.Error for this? Using Red was an error on the main branch. This has been fixed in v0.13.0-pre.

Comment (fishfishfish2104) [reply cmd/system.go]: sure ill change it now

**Comment by fishfishfish2104** [comment]
gonna close this pull request and open a new one as I cant seem to change the branch

**Pull request closed without merging**